src/requirements/data/majors/cs.ts

```diff
@@ -83,6 +80,23 @@ const csRequirements: readonly CollegeOrMajorRequirement[] = [
     checker: null,
     operator: null,
     fulfilledBy: 'self-check'
+  },
+  // TODO: Requirement whose classes can be double counted
```

Comment:
Do we want to include this rn even tho we don't have double counting? Either way, make an item

Reply:
We do now. I believe we can just remove that TODO

Reply:
jokes. Made monday item